### PR TITLE
Two batch files to run map merge.

### PR DESCRIPTION
**Pull request by xxalpha**
**Run Map Merge - TGM.bat** for all station maps, their z levels and generic z levels.
**Run Map Merge - DMM.bat** for all maps that are loaded at runtime (templates, away missions).

Until someone changes the map loader to read tgm, this will do.

### CODE REVIEW

**Comment by KazeEspada** [comment]
.bat files. We really hate linux users here.

**Comment by Iamgoofball** [comment]
minority

**Comment by xxalpha** [comment]
Linux users can make their own :^)

**Comment by as334** [comment]
Yes. Just not actual byond. Any yeah they should have Wine if they're playing spessmen anyways.

**Comment by duncathan** [comment]
A Linux user is smart enough to make their own file to run the thing, jeez

File: tools/mapmerge/map_helpers.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def merge_map(newfile, backupfile, tgm):
         write_grid_coord_small(newfile, mergeGrid)
     else:
         write_dictionary(newfile, originalDict)
-        write_grid(newfile, originalDict)
+        write_grid(newfile, mergeGrid)
```
Comment (duncathan) [inline tools/mapmerge/map_helpers.py]: This is intended?

Comment (xxalpha) [inline tools/mapmerge/map_helpers.py]: Yea that's how it was supposed to be. It's currently bugged if you want to write a normal dmm map.

**Comment by xxalpha** [comment]
I would make the linux files myself, but since I can't test them I don't. It's easy for someone to add them at any time.

**Comment by optimumtact** [comment]
hah

**Comment by Buggy123** [comment]
This doesn't work in linux, lemme get the error code...

**Comment by optimumtact** [comment]
uh mate 
[ 114.589823] EXT4-fs error (device mccblk0p2): ext_lookup:1588: inode #96857: comm python3.5: deleted inode referenced: 96858

your file system is corrupted, you should probably run an fsck

**Pull request merged**